### DISCUSSION TITLE
Model geometry is missing in AR view

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#3518] [Q&A] [answered]
When entering AR view some of the model geometry is missing from the model. The model appears perfectly fine in the 3D view. The issue happens on my iPhone 13 pro max.

I'm currently using RGB values to control the base colour, would this affect the models geometry?  

Link: https://inplaceclient.s3.ap-southeast-2.amazonaws.com/domusliving/close+/RGB+testing+5.html 

Browser Affected
- [ ] Chrome, - Tested on pixel 6 pro - works perfectly fine 
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ x ] Safari  - tested on iPhone 13 pro max 
![282447883_702423174317251_1669198711250972270_n](https://user-images.githubusercontent.com/90825510/172295749-eee69e36-ea9b-4ddc-9b98-abd90f5c6157.jpg)

**Comment by elalish** [comment]
@mrdoob I can't remember: does USDZExporter handle applying a baseColorFactor to the texture, or does that just get lost in translation?

**Comment by mrdoob** [comment, marked as answer]
The problem here is that the model is double sided.

And I don't think USD supports double sided...
https://developer.apple.com/forums/thread/112662

If we can confirm that we could make `USDZExporter` log a warning in these cases.
